metadata_backend/helpers/schemas/BP.schema.xsd

```diff
@@ -3,9 +3,11 @@
     <xs:import schemaLocation="BP.annotation.xsd"/>
     <xs:import schemaLocation="BP.dataset.xsd"/>
     <xs:import schemaLocation="BP.image.xsd"/>
+    <xs:import schemaLocation="BP.observation.xsd"/>
     <xs:import schemaLocation="BP.sample.xsd"/>
+    <xs:import schemaLocation="BP.staining.xsd"/>
     <xs:import schemaLocation="EGA.dac.xsd" namespace="SRA"/>
     <xs:import schemaLocation="EGA.policy.xsd" namespace="SRA"/>
     <xs:import schemaLocation="SRA.study.xsd" namespace="SRA"/>
-    <xs:import schemaLocation="SRA.submission.xsd" namespace="SRA"/>
+    <xs:import schemaLocation="BP.submission.xsd" namespace="SRA"/>
```

Comment:
Is this correct?

Reply:
Right. This `BP.schema.xsd` is just an xsd file that Bigpicture has implemented. I'm not entirely sure of its relevance in our schema directory.

Reply:
And so they have their own submission schema but to my knowledge we don't need to implement it here.

Reply:
But yea, I'm pretty sure we can remove this schema